### PR TITLE
add testing and add errorless eslint

### PR DESCRIPTION
**Pull request by fabianovangool**

### CODE REVIEW

File: src/components/member-modal/MemberModal.test.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { render, screen, fireEvent } from "@testing-library/react";
+import { MemberModal } from "./MemberModal";
+
+describe("Close member modal", () => {
+  const onClose = jest.fn();
+  const handleAddButtonClick = jest.fn();
+
+  it("Press cancel, onClose gets called", () => {
```
Comment (RubenWerdmuller) [inline src/components/member-modal/MemberModal.test.tsx:8]: the `it press cancel` grammer is a bit odd to read ;)

**Review by brambrink — COMMENTED** [review]
Looking very nice! 🆒

File: src/components/member-modal/MemberModal.tsx [hunk 2]
```diff
@@ -44,6 +44,7 @@ export const MemberModal: React.FC<Types.MemberModalProps> = ({
 
   return isActive
     ? createPortal(
+        // eslint-disable-next-line react/jsx-indent
```
Comment (RubenWerdmuller) [inline src/components/member-modal/MemberModal.tsx:47]: Usually, you would fix this here or through ESLint

**Review by brambrink — APPROVED** [review]
Looks good!

File: src/components/member-entries/MemberEntries.test.tsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
+import { waitFor } from "@testing-library/react";
+import { addMemberEntry } from "../../services/team-members-api";
+
+const mockedNewTeamMember = {
+  firstName: "Enzo",
+  lastName: "Ferrari",
```
Comment (amijsb) [inline src/components/member-entries/MemberEntries.test.tsx:6]: Nice 🏎

File: .github/workflows/build-app.prod.yml [hunk 4]
```diff
@@ -19,3 +19,30 @@ jobs:
 
       - name: 📦  build
         run: npm run build
+
+      - name: 🧹 Run lint
+        run: npm run lint
```
Comment (amijsb) [inline .github/workflows/build-app.prod.yml:24]: I'd run `npm run lint` before running `npm run build`.

File: src/components/time-entry/TimeEntry.tsx [hunk 5]
```diff
@@ -40,7 +41,10 @@ export const TimeEntry = ({ id, client, startTimestamp, stopTimestamp }: Types.T
           <Styled.WorkTime>{formattedWorkHours}</Styled.WorkTime>
         </Styled.TotalWorkHours>
       </Styled.LocationHoursWrapper>
-      <Styled.TrashButton onClick={handleDelete}>
+      <Styled.TrashButton
+        // eslint-disable-next-line react/jsx-no-bind
```
Comment (amijsb) [inline src/components/time-entry/TimeEntry.tsx:45]: Do you need this eslint exception?

File: src/components/time-entries/TimeEntries.tsx [hunk 6]
```diff
@@ -123,6 +123,7 @@ export const TimeEntries = (props: TimeEntriesProps) => {
         <Modal
           isActive={isModalActive}
           onClose={() => setIsModalActive(false)}
+          // eslint-disable-next-line react/jsx-no-bind
```
Comment (amijsb) [inline src/components/time-entries/TimeEntries.tsx:126]: Do you need this eslint exception?

Comment (amijsb) [reply src/components/time-entries/TimeEntries.tsx:126]:
```suggestion
         handleAddButtonClick={() => handleClick()}
```

File: src/components/time-entry/TimeEntry.tsx (hunk 5 above)
Comment (amijsb) [reply src/components/time-entry/TimeEntry.tsx:45]:
```suggestion
        onClick={() => handleDelete()}
```

**Review by amijsb — CHANGES_REQUESTED** [review]
Good busy 🚀

**Review by amijsb — APPROVED** [review]
🏎🏎🏎💨💨💨

**Pull request merged**